numba_rvsdg/core/datastructures/ast_transforms.py

```diff
@@ -655,11 +666,220 @@ def render(self) -> None:
+            def codegen_view() -> list[Any]:
+                return [
+                    self.codegen(b)
+                    for b in block.subregion.concealed_region_view.values()  # type: ignore  # noqa
+                    if not (type(b) is RegionBlock and b.kind == "branch")
+                ]
+
+            # Head, tail and branch regions themselves to use the custom view
+            # above.
+            if block.kind == "head":
+                rval = codegen_view()
+            elif block.kind == "tail":
+                rval = codegen_view()
+            elif block.kind == "branch":
+                rval = codegen_view()
+            elif block.kind == "loop":
+                # A loop region gives rise to a Python while True loop. We
+                # recursively visit the body.
+                rval = [
+                    ast.While(
+                        test=ast.Constant(value=True),
+                        body=codegen_view(),
+                        orelse=[],
+                    )
+                ]
+            else:
+                raise NotImplementedError
+            self.region_stack.pop()
+            return rval
+        elif type(block) is SyntheticAssignment:
+            # Synthetic assignments just create Python assignments, one for
+            # each variable..
+            return [
+                ast.Assign([ast.Name(t)], ast.Constant(v), lineno=0)
+                for t, v in block.variable_assignment.items()
+            ]
+        elif type(block) is SyntheticTail:
+            # Synthetic tails do nothing.
+            pass
+        elif type(block) is SyntheticFill:
+            # Synthetic fills must have a pass statement to main syntactical
+            # correctness of the final program.
+            return [ast.Pass()]
+        elif type(block) is SyntheticReturn:
+            # Synthetic return blocks must re-assigne the return value to a
+            # special reserved variable.
+            return [ast.Return(ast.Name("__return_value__"))]
+        elif type(block) is SyntheticExitingLatch:
+            # The synthetic exiting latch much create a query on the variable
+            # it holds and then insert a Python if that will either break or
+            # continue. This effectively generates the backedge for the looping
+            # region.
+            assert len(block.jump_targets) == 1
+            assert len(block.backedges) == 1
+            compare_value = [
+                i
+                for i, v in block.branch_value_table.items()
+                if v == block.backedges[0]
+            ][0]
+            if_beak_node_test = ast.Compare(
+                left=ast.Name(block.variable),
+                ops=[ast.Eq()],
+                comparators=[ast.Constant(compare_value)],
+            )
+            if_break_node = ast.If(
+                test=if_beak_node_test,
+                body=[ast.Continue()],
+                orelse=[ast.Break()],
+            )
```

Comment:
I found a way to avoid the use of `continue` and `break`.
Instead of:

```python

while True:
    cont = <loop_body>
    if cont:
        continue
    else:
        break
```
emit this:
```python
cont = True
while cont:
    cont = <loopbody>
```

Reply:
you can try ea3ef44ac5201a67a519221ba2ed4b2003e4f09b